doc/code/qml_noise.rst

```diff
@@ -0,0 +1,90 @@
+qml.noise
+=========
+
+This module contains the functionality for building and manipulating noise models.
+
+Overview
+--------
+
+In PennyLane, noise models are defined via a mapping of conditions defined as ``Conditionals``
+to functions specified as ``Callables``, along with some additional noise-related metadata.
```

Comment:
We cannot directly use `quantum functions` here since their definition specifies having a return statement (for measurement. Added an extra line for it.

Reply:
It's an interesting point. We definitely say [here](https://docs.pennylane.ai/en/stable/introduction/circuits.html#quantum-functions) that a quantum function must return a measurement, but we don't really follow that throughout PL (and probably throughout the docs too).